app/Http/Requests/Api/Auth/RegisterRequest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Http\Requests\Api\Auth;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class RegisterRequest extends FormRequest
+{
+    public function rules()
+    {
+        return [
```

Comment:
Some of these fields are required like phone, date of birth and gender as mentioned here https://docs.google.com/spreadsheets/d/14u6iINxLTD4sh6Hev5lrChmpTzwWQa1bm7BWXLzedoE/edit#gid=0